sdk/containerregistry/azure-containerregistry/azure/containerregistry/_exchange_client.py

```diff
@@ -73,7 +74,10 @@ def get_refresh_token(self, service, **kwargs):
     def exchange_aad_token_for_refresh_token(self, service=None, **kwargs):
         # type: (str, Dict[str, Any]) -> str
         refresh_token = self._client.authentication.exchange_aad_access_token_for_acr_refresh_token(
-            service=service, access_token=self._credential.get_token(*self.credential_scopes).token, **kwargs
+            grant_type=PostContentSchemaGrantType.ACCESS_TOKEN,
```

Comment:
it seems that the service allows for users to pass in more than just "access_token" for grant_type, and it doesn't look like we're exposing this functionality to users. Is this expected?

Reply:
Yes. The grant_type is from generated code(new swagger), we are not exposing it to customers for now.